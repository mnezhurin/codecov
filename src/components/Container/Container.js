import React from 'react';

export const Container = ({ classNames = '', children}) => {
    return (
        <main className={classNames}>
            {children}
        </main>
    );
};
