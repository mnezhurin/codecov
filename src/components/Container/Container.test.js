import React from 'react';
import renderer from 'react-test-renderer';
import { Container } from './Container';

test('Container', () => {
    const component = renderer.create(
        <Container>
            <div>I am in the Container</div>
        </Container>,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});