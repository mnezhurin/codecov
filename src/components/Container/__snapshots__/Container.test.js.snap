// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container 1`] = `
<main
  className=""
>
  <div>
    I am in the Container
  </div>
</main>
`;
